### DISCUSSION TITLE
Is there any way to enable OpenAI-compatible provider for Gemini and Claude models?

### DISCUSSION THREAD

**Discussion by Astricaelus** [danny-avila/LibreChat#3341] [Help Wanted]
For Gemini and Claude models, sometimes third-party API providers (like OpenRouter) are used to call them. In such cases, the providers use OpenAI format interfaces instead of the native interfaces of these models. While it is possible to use "Custom Provider," it seems better if we could directly use OpenAI format interfaces within the builtin "Google Provider" and "Claude Provider."

**Comment by hiyukoim** [comment]
have you tried OpenRouter?
